src/core/extras/ntt2.cpp

```diff
@@ -196,10 +192,8 @@ void test_NTT(const usint level, const usint nloop) {
     ILParams params3(m, q3, rootOfUnity3);
     auto x3p = std::make_shared<ILParams>(params3);
 
-    Poly::DugType dug3;
-    dug3.SetModulus(q1);  // random # generator to use
```

Comment:
would like to make sure that dug3 doesn't need to be constructed with the modulus "q1".

Reply:
same